aspnet/fundamentals/app-state/sample/src/AppState/Model/RequestEntry.cs

```diff
@@ -1,11 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace AppState.Model
 {
-    [Serializable]
```

Comment:
@shahid-pk Thanks. So according to packagesearch it there is no serializable yet. Should we leave it broken for .NET Core? @ardalis